### PR TITLE
Resolve #48, Resolve #42: Merge of markdown-upload and markdown-frontend

### LINKED ISSUE #42: Format data as Markdown
We'll soon upload Markdown formatted data, which should be displayed accordingly.

### LINKED ISSUE #48: Convert Data to Markdown Before Upload
Convert all findings fields, except the snippets, on `publish findings`. Convert `description` and `fix.description` on `publish metadata`.

### PR DESCRIPTION
**Pull request by joschli**

### CODE REVIEW

File: mubench.reviewsite/templates/review.phtml [hunk 1]
```diff
@@ -240,30 +240,22 @@ use MuBench\ReviewSite\Model\Experiment;
...
+                    <object id="<?= $object_id ?>" type="image/svg+xml" data="<?= "{$uploads_url_prefix}/{$experiment->getId()}/{$detector->name}/{$misuse->getProject()}/{$misuse->getVersion()}/{$hit[$column]}" ?>" style="width: 300px; height: 300px;"></object>
                     <script>
                         document.getElementById("<?= $object_id ?>").onload = function() {
                             svgPanZoom("#<?= $object_id ?>", {
                                 zoomEnabled: true
                             });
                         };
                     </script>
-                <?php elseif(strlen($hit[$column]) > 4 && substr($hit[$column], -4) === ".png") : ?>
-                    <img src="<?= "{$uploads_url_prefix}/{$experiment->getId()}/{$detector->name}/{$misuse->getProject()}/{$misuse->getVersion()}/{$hit[$column]}" ?>" />
```
Comment (salsolatragus) [inline mubench.reviewsite/templates/review.phtml]: Reintroduce dedicated png handling.

File: mubench.reviewsite/style/style.css [hunk 2]
```diff
@@ -19,6 +19,7 @@ h2 {
 
 p {
 	margin: 10pt 0;
+    display: inline;
```
Comment (salsolatragus) [inline mubench.reviewsite/style/style.css]: Fix indent.

File: mubench.reviewsite/src/DBConnection.php [hunk 3]
```diff
@@ -126,6 +126,16 @@ public function getRuns(Detector $detector, $exp, $max_reviews = -1)
         return $runs;
     }
 
+    private function parseMarkdownInQuery($query){
```
Comment (salsolatragus) [inline mubench.reviewsite/src/DBConnection.php]: Delete obsolete method.

**Comment by joschli** [comment]
i made the changed, this can be reviewed again @salsolatragus

File: mubench.reviewsite/templates/review.phtml [hunk 4]
```diff
@@ -240,30 +239,24 @@ use MuBench\ReviewSite\Model\Experiment;
...
-                <?php if(strpos($hit[$column], ";") !== false) : ?>
-                    <ul>
-                        <?php foreach(explode(";", $hit[$column]) as $v) : ?>
-                            <li><?= htmlspecialchars($v); ?></li>
-                        <?php endforeach; ?>
-                    </ul>
-                <?php elseif(strlen($hit[$column]) > 4 && substr($hit[$column], -4) === ".svg") : ?>
-                    <? $object_id = $column . $hit['rank']; ?>
-                    <object id="<?= $object_id ?>" type="image/svg+xml" data="<?= "{$uploads_url_prefix}/{$experiment->getId()}/{$detector->name}/{$misuse->getProject()}/{$misuse->getVersion()}/{$hit[$column]}" ?>" style="width: 750px; height: 750px;"></object>
+                <?php if(strlen($hit[$column]) > 4 && in_array(substr($hit[$column], -4), [".png", ".svg"])) : ?>
```
Comment (salsolatragus) [inline mubench.reviewsite/templates/review.phtml]: This branch still capture the png case. Please adjust the condition.

File: mubench.reviewsite/templates/review.phtml [hunk 5]
```diff
@@ -240,30 +239,24 @@ use MuBench\ReviewSite\Model\Experiment;
...
+                    <object id="<?= $object_id ?>" type="image/svg+xml" data="<?= "{$uploads_url_prefix}/{$experiment->getId()}/{$detector->name}/{$misuse->getProject()}/{$misuse->getVersion()}/{$hit[$column]}" ?>" style="width: 300px; height: 300px;"></object>
                     <script>
                         document.getElementById("<?= $object_id ?>").onload = function() {
                             svgPanZoom("#<?= $object_id ?>", {
                                 zoomEnabled: true
                             });
                         };
                     </script>
-                <?php elseif(strlen($hit[$column]) > 4 && substr($hit[$column], -4) === ".png") : ?>
+                <?php elseif(strlen($hit[$column]) > 4 && substr($hit[$column], -4) === ".png"): ?>
```
Comment (salsolatragus) [inline mubench.reviewsite/templates/review.phtml]: Please add the missing whitespace.

**Pull request merged**